fineract-avro-schemas/src/main/avro/loan/v1/LoanSchedulePeriodDataV1.avsc

```diff
@@ -322,6 +322,22 @@
                 "null",
                 "bigdecimal"
             ]
+        },
```

Comment:
@josehernandezfintecheandomx Please remove this.

Reply:
Done!